manifests/base/mc-fleet-apps.yaml

```
@@ -0,0 +1,37 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: management-clusters-fleet-app
+  namespace: argocd
+spec:
+  project: management-clusters-fleet
```

Maybe this makes sense to make that more obvious and aligned with the build directory layout?

```suggestion
  project: management-clusters-fleet-argocd
```